environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
     - pooch
     - xarray
     - pandas
+    - rasterio
```

Comment:
This needs to be added to `requirements.txt`, `setup.py`, and `doc/install.rst`.

Reply:
Done!